stylesheets/_layout.settings.scss

```diff
@@ -468,3 +468,128 @@
 .content-actions__buttons .btn {
     vertical-align: top;
 }
+
+/**
+ * Stacked inputs
+ */
+ .tab__settings .settings-section--stacked {
+
+    .form__group {
+        clear: both;
+    }
+
+    // Force labels to be stacked with the inputs below, except for checkbox/radio
+    .form__group:not(.form__group--flush):not(.form__group--toggle):not(.form-checkbox):not(.form-radio) {
+        margin-bottom: 12px;
+
+        @include ie(9) {
+            &.colorpicker .form__control {
+                width: calc(100% - 48px);
+            }
+        }
+
+        > .control__label {
+            padding-top: 0;
+        }
+
+        > .control__label,
+        .controls,
+        .form__control:not(.checkbox):not(.radio),
+        .form__control + .select2,
+        .input-group {
+            width: 100%;
+        }
+
+        .help-block {
+            margin: 0;
+        }
+    }
+
+    // Because the settings sidebar and the toggle switch controls are a fixed width, we're safe to use a fixed width here
+    .form__group--toggle {
+        .controls {
+            float: left;
+
+            @include respond-min($screen-medium) {
+                float: right;
+            }
+        }
+
+        .control__label {
+            margin-left: 10px;
+            width: 80%;
+
+            @include respond-min($screen-smaller) {
+                width: 125px;
+            }
+
+            @include respond-min($screen-medium) {
+                margin-left: 0;
+                width: 250px;
+            }
+        }
+    }
+
+    // Instead of stacking, keep checkbox on left edge
+    .form__group.form-checkbox,
+    .form__group.form-radio {
+        position: relative;
+
+        > .control__label {
+            min-width: 300px;
+            padding-left: 30px;
+            width: 300px;
+        }
+
+        .controls {
+            left: 0;
+            min-width: 20px;
+            position: absolute;
+            top: 1px;
+            width: 20px;
+        }
+    }
+
+    .form-checkbox,
+    .form-radio {
+        margin-bottom: 12px;
+
+        > .control__label {
+            padding-top: 0;
+        }
+
+        .checkbox,
+        .radio {
+            top: 0;
+        }
+    }
+
+    .form-choice > .controls {
+        margin-top: 0;
+    }
+
+    .form-choice > .controls > .control__label {
+        width: 100%;
+    }
+}
+
+/**
+ * Sticky sidebar
+ */
+
+@media screen and (min-width: 992px) {
+    .tab__inner.is-sticky .tab__content {
+        left: calc(370px + 1%);
```

Comment:
What's the 1% here for?

Reply:
There's a 1% margin on the tab settings that this bases a calculation on

Reply:
👍